### ISSUE TITLE
Add document statistics

### ISSUE THREAD

**Issue by mike-ward** [mike-ward/Markdown-Edit#108]
Looking for ideas on this one. Here's a screen shot from another program.

![image](https://cloud.githubusercontent.com/assets/567927/12787519/bfc698c6-ca63-11e5-9aa2-488a187a9c69.png)

**Comment by cyotek** [comment]
If by ideas you mean feedback, I myself probably wouldn't use it as I don't really use MDE for writing formal documents or documents where I have min/max requirements on words etc. However if you did implement this, I think one potential issue is that the Grade Level and Fog Index (and assumably Reading Ease if that is derived from the former) seem to be American constructs which might not necessarily fit with other cultures.

In a nutshell, I might find some occasional value in the stats on the left, but not those on the right.

**Comment by borekb** [comment]
Probably not really useful but found this: https://www.npmjs.com/package/textstat
